edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/DockerDesiredModule.cs

```diff
@@ -16,9 +16,10 @@ public class DockerDesiredModule : DockerModule
             RestartPolicy restartPolicy,
             string type,
             DockerConfig settings,
+            PullPolicy pullPolicy,
```

Comment:
Did you test it with deployments where this is not specified?
This property will not be specified in deployments by default, and needs to default to IfNotPresent

Reply:
Yeah, if this isn't specified, the default was to use IfNotPresent. It is working with the current change set cause the default enum value for PullPolicy was IfNotPresent/Always. Did you mean making it nullable instead explicitly? I feel like that would make the code a bit uglier is all.

Reply:
Added a new default value of None to the PullPolicy enum so that in case a customer doesn't specify a value explicitly, the reported properties won't contain this property